apps/src/netsim/NetSimVisualization.js

```diff
@@ -273,8 +273,8 @@ NetSimVisualization.prototype.onRemoteChange_ = function () {
 NetSimVisualization.prototype.getElementByEntityID = function (elementType, entityID) {
   return _.find(this.elements_, function (element) {
     return element instanceof elementType &&
-        element.getCorrespondingEntityID &&
-        element.getCorrespondingEntityID() === entityID;
+        element.getCorrespondingEntityId &&
```

Comment:
The inheritance chain for stuff in the visualization looks something like this:

```
NetSimVizElement
  NetSimVizNode
    NetSimVizAutoDnsNode
    NetSimVizSimulationNode
  NetSimVizWire
    NetSimVizSimulationWire
```

Only the ones with `Simulation` in the name actually map to simulation entities - they are the only ones that implement `getCorrespondingEntityID`.  This is a place where I'd like to use an interface, but duck typing will have to suffice.

Things are inherited this way because `NetSimVizSimulationNode` actually has a whole lot more in common with `NetSimVizAutoDnsNode` than it does with `NetSimVizSimulationWire`.  There might be a good composition solution to this problem though.